.github/workflows/build-push-multi-arch.yml

```diff
@@ -21,11 +21,11 @@ on:
         required: true
         type: string
         default: 'Dockerfile'
-      architecture:
-        description: 'The desired architecture [`amd64` or `amd64 & arm64`]'
+      architectures:
+        description: 'The desired architecture(s) as comma-separated list (for example "amd64,arm64")'
         required: true
         type: string
-        default: 'amd64 & arm64'
+        default: 'amd64,arm64'
```

Comment:
comment: this is a sensible change. I was never quite happy with the `&` - comma separated makes much more sense.